src/highlight.js

```diff
@@ -70,7 +70,9 @@ const HLJS = function(hljs) {
     languages: null,
     // beta configuration options, subject to change, welcome to discuss
     // https://github.com/highlightjs/highlight.js/issues/1086
-    __emitter: TokenTreeEmitter
+    __emitter: TokenTreeEmitter,
+    // https://github.com/highlightjs/highlight.js/pull/3180
+    allCodeSelector: 'pre code',
```

Comment:
Let's go with `cssSelector` the scope of all these settings is implied to be all, they are global.